systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java

```diff
@@ -707,7 +707,7 @@ public void addFilter(FilterType filterType, String filterValue) throws Exceptio
                 selenium.clickOnItem(getStatusFilterDropDownItem());
                 selenium.clickOnItem(getSelectStatusDropDown(), "Status phase dropdown");
                 selenium.clickOnItem(getSelectStatusDropDown()
-                        .findElement(By.id("al-filter-select-status-dropdown-itemstatus" + filterValue.substring(0, 1).toUpperCase() + filterValue.substring(1))));
+                        .findElement(By.id("al-filter-select-status-dropdown-item" + filterValue.toLowerCase()})));
```

Comment:
@ajaypratap003 obviously on this line is and issue `}` char brake compilation... You should try compile your code before pushing.

Reply:
thanks @kornys for notice this typo issue. I"ll fix this.